db/migrate/20170125193809_add_planning_fields_to.rb

```diff
@@ -3,7 +3,7 @@
 class AddPlanningFieldsTo < ActiveRecord::Migration[6.0]
   def change
     add_column :employees, :planning_raise_date, :date
-    add_column :employees, :planning_raise_salary, :decimal
+    add_column :employees, :planning_raise_salary, :string
```

Comment:
Rather than changing the field to a string when the column was added, generate a new migration and do the change there.  There's some who say never change migrations in the past for any reason - I'm not one of them, I think there are some circumstances where you should go back and fix things, but hopefully those are few and far between.

In order to get this change into production, you'd need to roll back to and rerun this migration.  Depending on what happened across all of those migrations, that could have bad consequences for our data.  Better to let this stay decimal in this migration and make a new one that switches the column over.

Reply:
@aterrype: Yes, @wubr's correct. Please undo this change and create a new migration that migrates the data in parallel with changing the schema.
All new migrations automatically run on deployment. All previously existing migrations are ignored, and assumed already run. A new migration that changes the structure, and (importantly!) migrates the existing data from the old to the new structure without losing it, is automatically run for every other database in play as the new code is pulled in. This works for production, staging, my development database, Jacob's database, as well as the database of any other Bendyworker who worked on this project in the past who jumps back in.
Migrations are such an elegant, beautiful solution to the previous norm of "all our databases have differing schemas". 
😄